stock_picking_quick/models/product_product.py

```diff
@@ -0,0 +1,42 @@
+# © 2019 Today Akretion
+# @author Pierrick Brun <pierrick.brun@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+from odoo import api, fields, models
+
+
+class ProductProduct(models.Model):
+    _inherit = 'product.product'
+
+    move_ids = fields.One2many(
```

Comment:
@PierrickBrun `string` is missing

Reply:
@nikul-serpentcs this is technical field (never displayed)

Reply:
Indeed @rousseldenis , I've added on still, it doesn't cost much